css-brochure-form/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Brochure Form</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <h3>LEARN MORE</h3>
+  <div class="main">
+    <form>
+      <label for="email">COMPANY EMAIL</label><br>
+      <input type="text" id="email" placeholder="sally@company.co"><br>
+      <label for="name">NAME</label><br>
+      <input type="text" id="name" placeholder="Sally Smith"><br>
+      <label for="location">WHERE ARE YOU HIRING?</label><br>
+      <input type="text" id="location" placeholder="City, Country"><br>
+      <label for="engineers">HOW MANY ENGINEERS ARE YOU HIRING IN THE NEXT 6 MONTHS?</label><br>
+      <select name="engineers" id="engineers">
+        <option value="1-2 Engineers">1-2 Engineers</option>
+      </select><br>
```

Comment:
Instead of using `br` elements, try using `div` elements or `display: block` to make elements go to the next line.